{"message":"500 - \"{\\\"Error\\\":{\\\"errors\\\":{\\\"languages.2.mediaContext.paragraphOne\\\":{\\\"name\\\":\\\"ValidatorError\\\",\\\"message\\\":\\\"Path `paragraphOne` (`Trouver les bonnes chaussures ...`, length 218) is longer than the maximum allowed length (205).\\\",\\\"properties\\\":{\\\"message\\\":\\\"Path `paragraphOne` (`Trouver les bonnes chaussures ...`, length 218) is longer than the maximum allowed length (205).\\\",\\\"type\\\":\\\"maxlength\\\",\\\"maxlength\\\":205,\\\"path\\\":\\\"paragraphOne\\\",\\\"value\\\":\\\"Trouver les bonnes chaussures ...\\\",\\\"length\\\":218},\\\"kind\\\":\\\"maxlength\\\",\\\"path\\\":\\\"paragraphOne\\\",\\\"value\\\":\\\"Trouver les bonnes chaussures ...\\\"},\\\"languages.2.mediaContext.paragraphTwo\\\":{\\\"name\\\":\\\"ValidatorError\\\",\\\"message\\\":\\\"Path `paragraphTwo` (`Pensez au amorti, à la stabili...`, length 228) is longer than the maximum allowed length (205).\\\",\\\"properties\\\":{\\\"message\\\":\\\"Path `paragraphTwo` (`Pensez au amorti, à la stabili...`, length 228) is longer than the maximum allowed length (205).\\\",\\\"type\\\":\\\"maxlength\\\",\\\"maxlength\\\":205,\\\"path\\\":\\\"paragraphTwo\\\",\\\"value\\\":\\\"Pensez au amorti, à la stabili...\\\",\\\"length\\\":228},\\\"kind\\\":\\\"maxlength\\\",\\\"path\\\":\\\"paragraphTwo\\\",\\\"value\\\":\\\"Pensez au amorti, à la stabili...\\\"}},\\\"_message\\\":\\\"Article validation failed\\\",\\\"name\\\":\\\"ValidationError\\\",\\\"message\\\":\\\"Article validation failed: languages.2.mediaContext.paragraphOne: Path `paragraphOne` (`Trouver les bonnes chaussures ...`, length 218) is longer than the maximum allowed length (205)., languages.2.mediaContext.paragraphTwo: Path `paragraphTwo` (`Pensez au amorti, à la stabili...`, length 228) is longer than the maximum allowed length (205).\\\"}}\"","name":"AxiosError","stack":"AxiosError: Request failed with status code 500\n    at settle (/usr/local/lib/node_modules/n8n/node_modules/.pnpm/axios@1.12.0/node_modules/axios/lib/core/settle.js:19:12)\n    at RedirectableRequest.handleResponse (/usr/local/lib/node_modules/n8n/node_modules/.pnpm/axios@1.12.0/node_modules/axios/lib/adapters/http.js:565:9)\n    at RedirectableRequest.emit (node:events:531:35)\n    at RedirectableRequest._processResponse (/usr/local/lib/node_modules/n8n/node_modules/.pnpm/follow-redirects@1.15.11/node_modules/follow-redirects/index.js:409:10)\n    at ClientRequest.RedirectableRequest._onNativeResponse (/usr/local/lib/node_modules/n8n/node_modules/.pnpm/follow-redirects@1.15.11/node_modules/follow-redirects/index.js:102:12)\n    at Object.onceWrapper (node:events:634:26)\n    at ClientRequest.emit (node:events:519:28)\n    at HTTPParser.parserOnIncomingClient [as onIncoming] (node:_http_client:716:27)\n    at HTTPParser.parserOnHeadersComplete (node:_http_common:117:17)\n    at TLSSocket.socketOnData (node:_http_client:558:22)\n    at TLSSocket.emit (node:events:519:28)\n    at addChunk (node:internal/streams/readable:561:12)\n    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)\n    at TLSSocket.Readable.push (node:internal/streams/readable:392:5)\n    at TLSWrap.onStreamRead (node:internal/stream_base_commons:189:23)\n    at TLSWrap.callbackTrampoline (node:internal/async_hooks:130:17)\n    at Axios.request (/usr/local/lib/node_modules/n8n/node_modules/.pnpm/axios@1.12.0/node_modules/axios/lib/core/Axios.js:45:41)\n    at processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at invokeAxios (/usr/local/lib/node_modules/n8n/node_modules/.pnpm/n8n-core@file+packages+core_@opentelemetry+api@1.9.0_@opentelemetry+sdk-trace-base@1.30_08b575bec2313d5d8a4cc75358971443/node_modules/n8n-core/src/execution-engine/node-execution-context/utils/request-helper-functions.ts:313:10)\n    at proxyRequestToAxios (/usr/local/lib/node_modules/n8n/node_modules/.pnpm/n8n-core@file+packages+core_@opentelemetry+api@1.9.0_@opentelemetry+sdk-trace-base@1.30_08b575bec2313d5d8a4cc75358971443/node_modules/n8n-core/src/execution-engine/node-execution-context/utils/request-helper-functions.ts:695:20)\n    at Object.request (/usr/local/lib/node_modules/n8n/node_modules/.pnpm/n8n-core@file+packages+core_@opentelemetry+api@1.9.0_@opentelemetry+sdk-trace-base@1.30_08b575bec2313d5d8a4cc75358971443/node_modules/n8n-core/src/execution-engine/node-execution-context/utils/request-helper-functions.ts:1749:4)","code":"ERR_BAD_RESPONSE","status":500}